drivers/pwm/pwm_stm32.c

```diff
@@ -103,6 +89,20 @@ static const uint32_t ch2ll[TIMER_MAX_CH] = {
 #endif
 };
 
+#if defined(LL_TIM_CHANNEL_CH1N)
```

Comment:
What about moving this #ifdef inside the array, so that array exist but has empty length if LL_TIM_CHANNEL_CH1N is not defined?
This remove the need to use `#ifdef LL_TIM_CHANNEL_CH1N` later in `pwm_stm32_pin_set()'

Reply:
done